video/external/Cargo.toml

```diff
@@ -17,11 +17,11 @@ ruffle_video_software = { path = "../software" }
 
 # Needed for OpenH264:
 libloading = "0.8.5"
-md-5 = "0.10.6"
 reqwest = { version = "0.12.5", default-features = false, features = ["blocking"] }
 hex = "0.4.3"
 bzip2 = { version = "0.4.4", features = ["static"] }
 tempfile = "3.11.0"
+sha2 = "0.10.8"
 
 [package.metadata.cargo-machete]
```

Comment:
I think this section can be removed now.

Reply:
Removed!